test/components/radiosfield.test.js

```diff
@@ -86,11 +90,16 @@ lab.experiment('RadiosField', () => {
       })
       lab.test('filter is not array', ({ context }) => {
         const viewModel = context.radiosField.getViewModel({ [componentName]: { filter: 1 } }, formData)
-        expect(viewModel.items.length).to.equal(3)
+        expect(viewModel.items.length).to.equal(4)
       })
       lab.test('filter has fewer than 2 valid items', ({ context }) => {
         const viewModel = context.radiosField.getViewModel({ [componentName]: { filter: [1, 99] } }, formData)
+        expect(viewModel.items.length).to.equal(4)
+      })
+      lab.test('includes divider items if not at the end of the list', ({ context }) => {
```

Comment:
Although it's implicitly covered in earlier tests, I would like to see an explicit negative test here and also one catering for when the divider is first in the list.